hvpy/utils.py

```diff
@@ -127,3 +129,23 @@ def create_events(events: List[Union[EventType, str, tuple]]) -> str:
             raise ValueError(f"{event} is not a EventType or str or two-length tuple")
     # Strips the final comma
     return constructed_events[:-1]
+
+
+def save_file(data: bytearray, filename: Path, overwrite: bool = False) -> None:
```

Comment:
```suggestion
def save_file(data: bytearray, filename: Union[Path, str], overwrite: bool = False) -> None:
```
I think you will need to import Union